src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      My name is
+      {' '}
+      {person.name}
+    </h2>
+
+    {person.age ? (
+      <p className="Person__age">
+        I am
+        {' '}
+        {person.age}
+      </p>
+    ) : (
+      null
+    )}
+
+    <p className="Person__partner">
+      {person.isMarried
+        ? (`${person.partnerName} is my ${person.sex === 'f' ? ('husband') : ('wife')}`
+        ) : (
+          'I am not married'
```

Comment:
Create a variable for husband/wife
![image](https://github.com/mate-academy/react_person/assets/96409563/47259a66-7c09-4d55-bad4-6e634aa00530)